[0m[[0m[0mdebug[0m] [0m[0m> Exec(run, Some(757790e2-cc2b-4b56-bef7-68556304143d), Some(CommandSource(console0)))[0m
[0m[[0m[0mdebug[0m] [0m[0mEvaluating tasks: Compile / run[0m
[0m[[0m[0mdebug[0m] [0m[0mRunning task... Cancel: Signal, check cycles: false, forcegc: true[0m
[0m[[0m[0minfo[0m] [0m[0mcompiling 1 Scala source to C:\Users\king\Desktop\BigDataModule\Project1\target\scala-2.11\classes ...[0m
[0m[[0m[33mwarn[0m] [0m[0mthere were 5 deprecation warnings; re-run with -deprecation for details[0m
[0m[[0m[33mwarn[0m] [0m[0mone warning found[0m
[0m[[0m[31merror[0m] [0m[0mjava.sql.SQLException: Column Index out of range, 4 > 1. [0m
[0m[[0m[31merror[0m] [0m[0m	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)[0m
[0m[[0m[31merror[0m] [0m[0m	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)[0m
[0m[[0m[31merror[0m] [0m[0m	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)[0m
[0m[[0m[31merror[0m] [0m[0m	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)[0m
[0m[[0m[31merror[0m] [0m[0m	at com.mysql.cj.jdbc.result.ResultSetImpl.checkColumnBounds(ResultSetImpl.java:494)[0m
[0m[[0m[31merror[0m] [0m[0m	at com.mysql.cj.jdbc.result.ResultSetImpl.getString(ResultSetImpl.java:871)[0m
[0m[[0m[31merror[0m] [0m[0m	at P1$.login(P1.scala:117)[0m
[0m[[0m[31merror[0m] [0m[0m	at P1$.main(P1.scala:42)[0m
[0m[[0m[31merror[0m] [0m[0m	at P1.main(P1.scala)[0m
[0m[[0m[31merror[0m] [0m[0m	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)[0m
[0m[[0m[31merror[0m] [0m[0m	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)[0m
[0m[[0m[31merror[0m] [0m[0m	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.lang.reflect.Method.invoke(Unknown Source)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run.invokeMain(Run.scala:143)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run.execute$1(Run.scala:93)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run.$anonfun$runWithLoader$5(Run.scala:120)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run$.executeSuccess(Run.scala:186)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run.runWithLoader(Run.scala:120)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Defaults$.$anonfun$bgRunTask$6(Defaults.scala:1983)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Defaults$.$anonfun$termWrapper$2(Defaults.scala:1922)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.util.Try$.apply(Try.scala:213)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.internal.BackgroundThreadPool$BackgroundRunnable.run(DefaultBackgroundJobService.scala:369)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.lang.Thread.run(Unknown Source)[0m
[0m[[0m[31merror[0m] [0m[0m(Compile / [31mrun[0m) java.sql.SQLException: Column Index out of range, 4 > 1. [0m
[0m[[0m[31merror[0m] [0m[0mTotal time: 38 s, completed Jan 31, 2022 1:37:50 PM[0m
[0m[[0m[0mdebug[0m] [0m[0m> Exec(shell, None, None)[0m
